lzy/allocator/src/main/java/ai/lzy/allocator/vmpool/yc/YcMk8s.java

```diff
@@ -152,7 +156,7 @@ public String getClusterPodsCidr(String clusterId) {
     // TODO: getters for YC-specific data
 
     private void resolveCluster(String clusterId, boolean system) {
-        LOG.info("Resolve {} cluster {}...", ct(system), clusterId);
+        LOG.debug("Resolve {} cluster {}...", ct(system), clusterId);
```

Comment:
this is very useful information, we don't want to lose it

Reply:
hmmm.... previously period was 10m :)
ok, once per minute -- it would be too much

let's then move all the messages in the debug, where nothing has changed

sorry :)